config.json

```diff
@@ -83,7 +83,12 @@
       "core": true,
       "difficulty": 3,
       "slug": "scrabble-score",
-      "topics": null,
+      "topics": [
+        "loops",
+        "pattern_matching",
+        "strings",
+        "transforming"
```

Comment:
Great suggestions! :) I think I would also add `games` as a topic since this is to do with scrabble. Most of the exercises this exercise unlocks are also to do with games (e.g. `minesweeper` and `queen-attack`) so it would be nice if they all have the `games` tag so people can see how they're related :)